test/Chirp.Infrastructure.Tests/RepositoryTests/AuthorRepositoryTest.cs

```diff
@@ -0,0 +1,261 @@
+using Chirp.Infrastructure.Repository;
+using Chirp.Core.Entities; 
+using Chirp.Infrastructure;
+using Test_Utilities;
+
+namespace Chirp.InfrastructureTests.RepositoryTests;
+
+public class AuthorRepositoryTest
+{
+
+    private readonly ChirpDbContext context;
+
+    public AuthorRepositoryTest()
+    {
+        context = SqliteInMemoryBuilder.GetContext();
+    }
+
+    [Fact]
+    public void GetAuthorByName_ShouldReturnCorrectAuthorDTO()
+    {
+        var authorRepository = new AuthorRepository(context);
+
+        AuthorDTO theAuthor = null;
+        for (int i = 0; i < 34; i++)
+        {
+
+            if (i == 5)
```

Comment:
Er der en speciel årsag til '5'?

Reply:
Primært for at vælge det midterste objekt, så vi garanterer at det ikke bare var første / sidste der blev valgt.
Nu nummer '1' i stedet (ud af 3)